public-api/vx/node.php

```diff
@@ -28,6 +28,17 @@
 		'grade-06-out' => ['length' => 1],
 		'grade-07-out' => ['length' => 1],
 		'grade-08-out' => ['length' => 1],
+		
+		'link-01-name' => ['length' => 64],
+		'link-02-name' => ['length' => 64],
+		'link-03-name' => ['length' => 64],
+		'link-04-name' => ['length' => 64],
+		'link-05-name' => ['length' => 64],
+		'link-01-url' => ['length' => 128],
```

Comment:
Gosh, I wish we could limit URLs to 128 characters, but it'll have to be longer. I need to do a quick check first, but I'm guessing `2048` will cover us. Also it should be `link-01` without the `-url` (as the `-name` feature will become less and less needed as we populate our platform tag list). ;)